CHANGELOG.rst

```diff
@@ -12,6 +12,16 @@ Change Log
 Unreleased
 ----------
 
+[7.1.0] - 2021-09-10
+--------------------
+
+Changed
+~~~~~~~
+
+* Updated ``EnsureJWTAuthSettingsMiddleware`` to understand and work work permissions combined using DRF's in-built support. 
```

Comment:
This sentence has two `work` in it.
Also, the structure is a little bit ambiguous.